run-clang-format.sh

```diff
@@ -47,3 +47,9 @@ for DIR in ${DIRS}; do
             -exec ${CLANG_FORMAT} -i -style=file {} +
     done
 done
+
+SOURCES="decree-cgc-cfe/fs/binfmt_cgc.c linux-4.9.3/fs/binfmt_elf.c"
+
+for SOURCE in $SOURCES; do
```

Comment:
Why?

Reply:
I found it easier to use clang-format than having to configure my editor for tabs instead of spaces.